src/util/parse_options.cpp

```diff
@@ -116,19 +117,23 @@ int parse_options_baset::main()
   }
 }
 
+std::string align_center_with_border(const std::string &text) {
+  auto const total_length = std::size_t{63};
+  auto const border = std::string{"* *"};
+  auto const fill = total_length - 2 * border.size() - text.size();
+  auto const fill_right = fill / 2;
+  auto const fill_left = fill - fill_right;
+  return border + std::string(fill_left, ' ') + text +
+         std::string(fill_right, ' ') + border;
+}
+
 std::string
 banner_string(const std::string &front_end, const std::string &version)
 {
+  static_assert(CHAR_BIT == 8,
+                "we do make the fairly safe assumption of 8 bit bytes");
```

Comment:
#917 :-)

Reply:
Fair enough, I'll remove it. In this case the worst that can happen is odd numbers in the `-bits` section anyway :woman_shrugging:

Reply:
Oh, sorry, I did not necessarily want you to feel compelled to remove it. Was just meaning to say that there is existing work to clean this up.